infrastructure/docker/celo/start.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+mkdir -p /data/keystore
+cp init/keystore/* /data/keystore
+cp init/password.txt /data/password.txt
+[ ! -d "/data/celo" ] && geth --nousb --datadir=/data init init/genesis.json
+
+ geth --port 3000 --networkid 1101 --identity "somerandomidentity" \
+    --ws --wsaddr "0.0.0.0" --wsport "8546" --wsorigins "*" \
+    --rpc --rpcport "8545" --rpcaddr "0.0.0.0" --rpccorsdomain "" \
+    --rpcapi "db,ssh,miner,admin,eth,net,web3,personal" \
+    --wsapi "db,ssh,miner,admin,eth,net,web3,personal" \
+    --datadir /data --syncmode "fast" \
+    --mine --miner.threads 1 --nousb \
+    --unlock 0x2b2976824682233807a197081119da511af12f7a --password data/password.txt \
```

Comment:
I just wanted to confirm that we're unlocking just one account. Don't we want to unlock the rest of them?

Reply:
This is a special "Validator" account in a proof-of-stake mechanism which mines new blocks. I believe a single validator chain should be enough to test Celo locally. We got a `genesis.json` from a Celo team with one validator which is also has it's BLS key written in `extraData` [property](https://github.com/keep-network/keep-core/blob/docker-my-celo/infrastructure/docker/celo/init/genesis.json#L23).

Reply:
This one unlock is a must-have for the reason @dimpar mentioned.